From a5b5a75bb965a184ca0370b3fd64c34d33f9d3ce Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 12 Jul 2018 10:57:05 +0300
Subject: [PATCH] board: habana: fix incorrect quotes placement

Current quotes placement results in an incorrect parameters for the
build_and_insmod_habanalabs() function and thus the function's execution
does not work as expected failing the test.
Fix the placement, so the function will get its parameters correctly.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/remote_load_firmware b/board/habanalabs/common/remote_load_firmware
index 86efe7b..50eea74 100755
--- a/board/habanalabs/common/remote_load_firmware
+++ b/board/habanalabs/common/remote_load_firmware
@@ -39,7 +39,7 @@
 function remote_bi_habanalabs() {
 	# build and insmod the habanalabs driver
 	# this depends on successful copy of the .bashrc and .bash_aliases
-	ssh -t $host bash -ic 'build_and_insmod_habanalabs -p "cpu_enable=1 cpu_queues_enable=1"' | tee -a $result
+	ssh -t $host 'bash -ic "build_and_insmod_habanalabs -p \"cpu_enable=1 cpu_queues_enable=1\" -d"' | tee -a $result
 	local success=`grep -cE "^$success_str" $result`
 	[ $success -ge 1 ] && return 0;
 
